sample/SampleHost/Program.cs

```diff
@@ -13,24 +14,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            var config = new JobHostConfiguration();
-            config.Queues.VisibilityTimeout = TimeSpan.FromSeconds(15);
-            config.Queues.MaxDequeueCount = 3;
-            config.LoggerFactory = new LoggerFactory().AddConsole();
+            while (true)
```

Comment:
Will undo these changes - was using this create/dispose loop to verify my changes. Before the changes, each restart resulted in ServiceBus error logs being written, now the host restarts are all clean.